requirements.txt

```diff
@@ -4,6 +4,7 @@ pytest==7.1.3
 pytest-json-report==1.5.0
 pytest-split==0.9.0
 pytest-xdist==3.2.1
+pluggy==1.0.0
```

Comment:
I don't know why, in my local I'm downloading the 1.0.0 version, but in the gitlab when I execute 
"./build.sh -i runner" it's installing version "pluggy-1.5.0"
This causes failures:
`/go/src/github.com/DataDog/system-tests/venv/lib/python3.9/site-packages/_pytest/main.py:303: PluggyTeardownRaisedWarning: A plugin raised an exception during an old-style hookwrapper teardown.
Plugin: 1386284331[458](https://gitlab.ddbuild.io/DataDog/system-tests/-/jobs/626652661#L458)08, Hook: pytest_sessionfinish
OSError: cannot send (already closed?)
For more information see https://pluggy.readthedocs.io/en/stable/api_reference.html#pluggy.PluggyTeardownRaisedWarning
  config.hook.pytest_sessionfinish(`